docs/application-connector/docs/08-03-register-manage-services.md

```diff
@@ -128,3 +128,46 @@ A successful response returns the ID of the registered service:
     ```
     curl https://gateway.kyma.local:{NODE_PORT}/{APP_NAME}/v1/metadata/services/{YOUR_SERVICE_ID} --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -k
     ```
+
+
+## Register API with specification URL
+
+Application Registry allows you to pass API specification in form of specification URL.
+
+To register API with specification URL replace `api.spec` with `api.specificationUrl`. Be aware that `api.spec` has higher priority than `api.specificationUrl`.
```

Comment:
I would rephrase this sentence to state that if you provide both api.spec and api.specificationUrl then api.spec will be used.

Reply:
Done!